Tests/Form/Widget/BaseWidgetTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Tests\Form\Widget;
+
+use Symfony\Bridge\Twig\Extension\FormExtension;
+use Symfony\Bridge\Twig\Extension\TranslationExtension;
+use Symfony\Bridge\Twig\Form\TwigRenderer;
+use Symfony\Bridge\Twig\Form\TwigRendererEngine;
+use Symfony\Bridge\Twig\Tests\Extension\Fixtures\StubFilesystemLoader;
+use Symfony\Bundle\FrameworkBundle\Tests\Templating\Helper\Fixtures\StubTranslator;
+use Symfony\Component\Form\FormExtensionInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\Form\Test\TypeTestCase;
+
+/**
+ * Class BaseWidgetTest.
+ */
+abstract class BaseWidgetTest extends TypeTestCase
+{
+    /**
+     * @var FormExtensionInterface
+     */
+    private $extension;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setUp()
+    {
+        parent::setUp();
+
+        $rendererEngine = new TwigRendererEngine(array(
+            'formatter.html.twig',
+            'form_div_layout.html.twig',
+        ));
+
+        if (class_exists('Symfony\Component\Form\Extension\Core\Type\RangeType')) {
```

Comment:
Instead of checking if the (unrelated ? ) `Symfony\Component\Form\Extension\Core\Type\RangeType` class exists, check if one of the classes in the if…else statement exists to make your choice.